### PR TITLE
data-science-and-ml content update

### PR DESCRIPTION
**Pull request by vatudorache**

### CODE REVIEW

**Comment by AlexanderHodicke** [comment]
All is good from my side - awaiting approval by @oheimburger for anything code related

File: data-platform/data-science-and-ml/oracle-data-science-service/code-assets/ods-mlflow.auth.mysql.demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4609d257",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define Username and Password as ENV Variables\n",
+    "os.environ['MLFLOW_TRACKING_USERNAME'] = '<USERNAME_GOES_HERE>'\n",
+    "os.environ['MLFLOW_TRACKING_PASSWORD'] = '<PASSOWRD_GOES_HERE>'"
```
Comment (oheimburger) [inline data-platform/data-science-and-ml/oracle-data-science-service/code-assets/ods-mlflow.auth.mysql.demo.ipynb:93]: Typo. Should read PASSWORD not PASSOWRD.

File: data-platform/data-science-and-ml/oracle-data-science-service/OCI-setup-MLFlow-tracking-server.pdf
Comment (oheimburger) [inline data-platform/data-science-and-ml/oracle-data-science-service/OCI-setup-MLFlow-tracking-server.pdf:1]: Command lines show hostname and username. Please redact them.

File: data-platform/data-science-and-ml/oracle-data-science-service/Jobs-with-OCI-Data-Science-ADS-Operator-Control.pdf
Comment (oheimburger) [inline data-platform/data-science-and-ml/oracle-data-science-service/Jobs-with-OCI-Data-Science-ADS-Operator-Control.pdf:1]: Command lines show hostname and username. Please redact them.

File: data-platform/data-science-and-ml/oracle-data-science-service/Jobs-for-distributed-training-with-OCI-Data-Science.pdf
Comment (oheimburger) [inline data-platform/data-science-and-ml/oracle-data-science-service/Jobs-for-distributed-training-with-OCI-Data-Science.pdf:1]: Command lines show hostname and username. Please redact them.

**Review by oheimburger — CHANGES_REQUESTED** [review]

File: data-platform/data-science-and-ml/oracle-data-science-service/code-assets/ods-mlflow.auth.mysql.demo.ipynb (hunk 1 above)
Comment (vatudorache) [reply data-platform/data-science-and-ml/oracle-data-science-service/code-assets/ods-mlflow.auth.mysql.demo.ipynb]: Resolved.

File: data-platform/data-science-and-ml/oracle-data-science-service/OCI-setup-MLFlow-tracking-server.pdf
Comment (vatudorache) [reply data-platform/data-science-and-ml/oracle-data-science-service/OCI-setup-MLFlow-tracking-server.pdf:1]: Resolved.

File: data-platform/data-science-and-ml/oracle-data-science-service/Jobs-with-OCI-Data-Science-ADS-Operator-Control.pdf
Comment (vatudorache) [reply data-platform/data-science-and-ml/oracle-data-science-service/Jobs-with-OCI-Data-Science-ADS-Operator-Control.pdf:1]: Resolved.

File: data-platform/data-science-and-ml/oracle-data-science-service/Jobs-for-distributed-training-with-OCI-Data-Science.pdf
Comment (vatudorache) [reply data-platform/data-science-and-ml/oracle-data-science-service/Jobs-for-distributed-training-with-OCI-Data-Science.pdf:1]: Resolved.

File: data-platform/data-science-and-ml/oracle-machine-learning/OML4R-installation-on-Oracle-Database.pdf
[the remark this answers is not in the document]
Comment (vatudorache) [reply data-platform/data-science-and-ml/oracle-machine-learning/OML4R-installation-on-Oracle-Database.pdf:1]: Resolved.

**Review by oheimburger — CHANGES_REQUESTED** [review]
It looks like updates are not pushed into this PR. Please push them.

**Review by oheimburger — APPROVED** [review]

**Pull request closed without merging**